Clicker/Views/PollBuilder/AskQuestionCell.swift

```diff
@@ -69,14 +100,36 @@ class AskQuestionCell: UICollectionViewCell {
 
 }
 
-extension AskQuestionCell: UITextFieldDelegate {
+extension AskQuestionCell: UITextViewDelegate {
 
-    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+    func textView(_ textView: UITextView, shouldChangeTextIn range: NSRange, replacementText text: String) -> Bool {
         let maxLength = IntegerConstants.maxQuestionCharacterCount
-        let currentString: NSString = (textField.text ?? "") as NSString
-        let newString: NSString =
-            currentString.replacingCharacters(in: range, with: string) as NSString
+        let currentString: NSString = (textView.text ?? "") as NSString
+        let newString: NSString = currentString.replacingCharacters(in: range, with: text) as NSString
         return newString.length <= maxLength
     }
+    
+    func textViewDidBeginEditing(_ textView: UITextView) {
+        if textView.textColor == .clickerGrey2 {
+            textView.text = nil
+            textView.textColor = .black
+        }
+    }
+
+    func textViewDidChange(_ textView: UITextView) {
+        charCountLabel.text = "\(questionTextView.text.count)/120"
+        updateQuestionTextViewHeight()
+        updateEditable()
+        textDidChange()
+    }
+    
+}
+
+extension UITextView {
```

Comment:
ok lol last thing can you add a `// MARK - UITextView ` here to section out this

Reply:
oops my bad i guess dangerbot wants it to be `// MARK: - UITextView`